jenkins/databricks/build.sh

```diff
@@ -212,10 +234,23 @@ set_dep_jars()
     dep_jars[ANNOT]=${PREFIX_SPARK}--common--tags--tags-${HIVE_VER_STRING}__hadoop-${sw_versions[HADOOP]}_${SCALA_VERSION}_deploy.jar
     artifacts[CORE]="-DgroupId=org.apache.spark -DartifactId=spark-core_${SCALA_VERSION}"
     dep_jars[CORE]=${PREFIX_SPARK}--core--core-${HIVE_VER_STRING}__hadoop-${sw_versions[HADOOP]}_${SCALA_VERSION}_deploy.jar
+
     artifacts[HIVE]="-DgroupId=org.apache.spark -DartifactId=spark-hive_${SCALA_VERSION}"
-    dep_jars[HIVE]=${PREFIX_SPARK}--sql--hive--hive-${HIVE_VER_STRING}__hadoop-${sw_versions[HADOOP]}_${SCALA_VERSION}_deploy_shaded.jar
+    if [[ "$BASE_SPARK_VERSION" == "3.3.2" ]]; then
+      # TODO what is significance of 819592503 and is this going to change over time?
```

Comment:
Can/should we use a wildcard to see what is there?

Reply:
+1, we probably should file an issue to stop hard-coding versions. Simply scan the dir with patterns for dependencies should be enough. It seems the hardcoded versions would only be required to disambiguate artifacts with multiple versions. However, it should never happen for a single dir /databricks/spark/jars

Reply:
I looked at this, and it would involve rewriting parts of the bash script. I filed https://github.com/NVIDIA/spark-rapids/issues/8319 for this.